ArmRobot/Assets/Scripts/Camera/RandomPositionCamera.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RandomPositionCamera : MonoBehaviour
+{
+    public float changePositionCamera = 0.005f;
```

Comment:
Good that you chose a small number. Is this based on anything in the Tobin paper?

Reply:
No, but it is something close to it.